pkgs/by-name/na/navicat-premium/package.nix

```
@@ -0,0 +1,142 @@
+{
+  fetchurl,
+  appimageTools,
+  lib,
+  libGL,
+  glib,
+  glibc,
+  pango,
+  harfbuzz,
+  fontconfig,
+  libX11,
+  freetype,
+  e2fsprogs,
+  expat,
+  p11-kit,
+  libxcb,
+  libgpg-error,
+  stdenv,
+  cjson,
+  libxcrypt-legacy,
+  curl,
+  makeWrapper,
+  autoPatchelfHook,
+  libxkbcommon,
+  libselinux,
+  language ? "en",
+  isLite ? false,
+}:
+assert lib.elem language [
+  "en"
+  "cs"
+];
+assert lib.isBool isLite;
+let
+  pname = if isLite then "navicat-premium-lite" else "navicat-premium";
+  version = "17.1.1";
+  srcHashes = {
+    en = {
+      lite = "sha256-etKXXttd+bSQTyOyzbDx1mXhBH6xqQ1e+aqZIGgKcHs=";
+      full = "sha256-A+JSvzt2E/zq5rajjmRlNFkxUGyu33fAkkBYkTMHcYM=";
```

```suggestion
      full = "sha256-ndMzGzHQ8MBt/bPRZWmP/aRo5ALGkZftYjcRirL26bM=";
```
Think you may need to use a wayback URL here.